gateway-service-remoteconfig/src/main/java/org/apache/knox/gateway/service/config/remote/config/RemoteConfigurationRegistry.java

```diff
@@ -32,6 +32,8 @@
     private String keyTab;
     private boolean useKeyTab;
     private boolean useTicketCache;
+    /* If true ensures that the auth scheme used to create znodes is `auth` and not `sasl` */
+    private boolean isBackwardsCompatible;
```

Comment:
The field name should be simply `backwardCompatible` (the method name `boolean isBackwardCompatible()`)

Reply:
The `is` prefix is to indicate that this is a boolean, just a habit. The config variable exposed to users is `backwardCompatible`